blocks/library/table/index.js

```diff
@@ -27,10 +28,29 @@ registerBlockType( 'core/table', {
 			type: 'array',
 			source: children( 'table' ),
 			default: [
-				<tbody key="1">
-					<tr><td><br /></td><td><br /></td></tr>
-					<tr><td><br /></td><td><br /></td></tr>
-				</tbody>,
+				[
+					'tbody', {}, [
+						'tr', {}, [
+							'td', {}, [
+								'br', {},
```

Comment:
I've mentioned elsewhere before, but I think these `br`s are meant to have a bogus attribute; unsure why we want real `br`s in the markup of the table by default